thedavidmeister/estimate-work
# selenium drivers as deps

fixes #95 

selenium testing!

To test:

1. Run `$ boot tests-web` and watch it do its thing

2. Try messing with `logo-test` in `index_test.clj` to something that will fail and re-run the tests. They should fail.

Docs for the web driver can be found at https://github.com/semperos/clj-webdriver/wiki/Taxi-API-Documentation
